AVFoundation-CombineTests/TestSubscriber.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  TestSubscriber.swift
+//  AVFoundation-CombineTests
+//
+//  Created by József Vesza on 2020. 07. 24..
+//  Copyright © 2020. József Vesza. All rights reserved.
+//
+
+import Combine
+
+class TestSubscriber<T>: Subscriber {
+    typealias Input = T
+    typealias Failure = Never
+    
+    let demand: Int
+    let onComplete: ([T]) -> Void
+    
+    private var receivedValues: [T] = []
+    private var subscription: Subscription? = nil
+    
```

Comment:
```suggestion
    /// Initializer for this Object. This `Subscriber` can be created with an arbitrary `demand` and a closure to execute when a `Subscription` emits completion.
    ///
    /// - Parameters:
    ///   - demand: Customizable demand this `Subcriber` will request from a `Subscription`.
    ///   - onComplete: Closure that is executed when a `Subscription` emits completion. This Object will pass the values received from a `Subscription` as the parameter.
```

Reply:
Added a slightly tweaked version of this